modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -285,3 +285,52 @@ This gives you a backup of the original synchronization data.
 == Register {sll} Clients
 
 include::snippets/create_bootstrap_repo_register.adoc[]
+
+
+
+[[clients-sleses-el-migration]]
+== Migrate Extended Life Clients to {sll}
+
+If an extended life (EL) client such as RHEL and all clones (like {centos}, {almalinux}, {rocky}, and {oracle}) is already registered as a minion on {productname} and users wanted to migrate it to {sll} they can use a re-activation key to apply the activation key that drives the migration.
+
+For more information about re-activation keys, see xref:client-configuration:activation-keys.adoc#reactivation[].
+
+This procedure was tested on EL9, but for other versions, the workflow is the same.
+
+The re-activation key is per minion, and can be generated with the {webui} or using the API.
+For more information, see https://documentation.suse.com/suma/4.3/api/suse-manager/api/system.html#apidoc-system-obtainReactivationKey-loggedInUser-sid.
+
+To re-activate a client, the user can run the bootstrap script on the client and pass the re-activation key as an environment variable.  Example:
+
+----
+REACTIVATION_KEY=<KEY> ./bootstrap_liberate9.sh
+----
+
+////
+// FIXME
+// Do we need a procedure as follows? (ATM, it is comment and does not
+// appear in the documentation.)
+// What needs to be adjusted?
+.Procedure: Migrating an EL client to {sll} with a bootstrap script
+. In the {webui}, create a reactivation key for the existing client.
+  On the [guimenu]``System Details`` tab of the existing client click [guimenu]``Reactivation``.
+  Click [guimenu]``Generate New Key``, and make a note of the new key.
+. In the {webui}, navigate to menu:Salt[Keys], locate the Salt key associated with the existing client, and click btn:[delete].
```

Comment:
we dont' need to delete the key. In face I don't know what the side effect it can have